examples/mkl/mkl_test.cpp

```diff
@@ -0,0 +1,68 @@
+#include <oneapi/mkl.hpp>
+#include <spblas/spblas.hpp>
+
+#include <fmt/ranges.h>
+
+int main(int argc, char** argv) {
```

Comment:
I presume this isn't necessary anymore ?  or is it how you are testing things ?

Reply:
Correct, this is left over from our exercise at the workshop; I just removed the `/examples/mkl` directory.